compute/acceptance/job_test.go

```diff
@@ -17,6 +17,73 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestAwsAccJobsCreate(t *testing.T) {
+	if _, ok := os.LookupEnv("CLOUD_ENV"); !ok {
+		t.Skip("Acceptance tests skipped unless env 'CLOUD_ENV' is set")
+	}
+
+	client := common.NewClientFromEnvironment()
+	jobsAPI := NewJobsAPI(context.Background(), client)
+
+	jobSettings := JobSettings{
+		NewCluster: &Cluster{
+			NumWorkers:   2,
+			SparkVersion: "6.4.x-scala2.11",
+			SparkConf:    nil,
+			AwsAttributes: &AwsAttributes{
+				Availability: "ON_DEMAND",
+			},
+			NodeTypeID: "r3.xlarge",
+		},
+		NotebookTask: &NotebookTask{
+			NotebookPath: "/Users/sri.tikkireddy@databricks.com/demo-terraform/demo-notebook",
```

Comment:
replace user-based folders with `/tf-test`. storing emails in the code is not really GDPR-compliant :)